variables.tf

```
@@ -0,0 +1,197 @@
+variable "region" {
+  type = string
+  description = "aws region on which you want to create the role"
+}
+
+variable "api_key" {
+  type = string
+  description = "Your organization's API key on the StackGuardian Platform"
+}
+
+variable "org_name" {
+  type = string
+  description = "Your organization name on StackGuardian Platform"
+}
+
+variable "workflow_group_name" {
+  type = string
+  description = "StackGuardian workflow group name"
+}
+
+variable "account_number" {
+  type = number
+  description = "AWS account number"
+}
+
+variable "url" {
+  type = string
+  description = "URL of the identity provider"
+}
+
+variable "client_id" {
+  type = string
+  description = "client IDs (audiences) that identify the application registered with the OpenID Connect provider"
+  default = "https://api.app.stackguardian.io"
+}
+
+variable "role_name" {
+  type = string
+  description = "name of the aws role thats getting created"
+}
+
+variable "aws_policy" {
+  type = string
+  description = "ARN of aws policy"
+}
+
+variable "cloud_connector_name" {
+  type = string
+  description = "StackGuardian Cloud connector name"
+}
+
+variable "connector_type" {
+  type = string
+  description = "type of connector. You can select anyone of the following AWS_STATIC, AWS_RBAC, AWS_OIDC, AZURE_STATIC, AZURE_OIDC, GCP_STATIC"
+}
+
+###########################################
+# AWS Connector
+#
+# To set these variables, please use environment variables:
+# export "TF_VAR_aws_access_key=xxxx"
+# export "TF_VAR_aws_secret_key=your-aws-secret-key"
+###########################################
+
+variable "aws_access_key_id" {
+  type = string
+  description = "your AWS acoount access key"
+}
+
+variable "aws_secret_access_key" {
+  type = string
+  description = "your AWS account secret access key"
+}
+
+variable "aws_default_region" {
+  type = string
+  description = "any default region you want to set, for all your deployments"
+}
+
+###########################################
+# Azure Connector
+#
+# To set these variables, please use environment variables:
+# export "TF_VAR_client_id=xxxx-yyyy-rrrr-fff"
+# export "TF_VAR_client_secret=your-client-secret"
+# export "TF_VAR_tenant_id=your-tenant-id"
+###########################################
+
+variable "armTenantId" {
+  type = string
+  description = "your azure account tenant id"
+}
+
+variable "armSubscriptionId" {
+  type = string
+  description = "your azure subscription id"
+}
+
+variable "armClientId" {
+  type = string
+  description = "your azure client id"
+}
+
+variable "armClientSecret" {
+  type = string
+  description = "your azure client secret id"
```

As before, not the id but the secret
```suggestion
  description = "your azure client secret"
```